.test-conda-env-py3.yml

```diff
@@ -7,3 +7,5 @@ dependencies:
 - python=3
 - cantera
 - pip
+- jax
+- jaxlib
```

Comment:
```suggestion
```
`jax` should already bring this in automatically: https://github.com/conda-forge/jax-feedstock/blob/7ebe55eae01e0ea909e27134842ee225a7e0326d/recipe/meta.yaml#L26

Reply:
Removed in 8e620cd.